client/public/index.html

```diff
@@ -9,21 +9,143 @@
+        .links {
+          width: 62%;
+          display: flex;
+          justify-content: space-around;
+        }
+        .space {
+          border-radius: 50%;
+          background-color: #84a98c;
+        }
+        .space:hover {
+          background-color: #8f938c;
+        }
+        .all {
+          display: flex;
+          align-items: center;
+          flex-direction: column;
+        }
+        .box {
+          padding-top: 100px;
+          width: 100%;
+          display: grid;
+          grid-template-columns: repeat(auto-fit, minmax(350px, 1fr));
+          grid-gap: 15px;
+          margin: 0 auto;
+        }
+        .card {
+          position: relative;
+          width: 300px;
+          height: 350px;
+          background: #84a98c;
+          margin: 30px auto;
+          border-radius: 44px;
+          box-shadow: 0 2px 10px rgba(0, 0, 0, 0.2);
+        }
+        .card:before,
+        .card:after {
+          content: "";
+          position: absolute;
+          top: 0;
+          left: 0;
+          width: 100%;
+          height: 100%;
+          border-radius: 44px;
+          background: #cad2c5;
+          transition: 0.5s;
+          z-index: -1;
+        }
+        .card:hover:before {
+          transform: rotate(20deg);
+          box-shadow: 0 2px 20px rgba(0, 0, 0, 0.2);
+        }
+        .card:after {
+          transform: rotate(10deg);
+          box-shadow: 0 2px 20px rgba(0, 0, 0, 0.2);
+        }
+        .card .imgBx {
+          position: absolute;
+          top: 10px;
+          left: 10px;
+          bottom: 10px;
+          right: 10px;
+          border-radius: 44px;
+          background: #222;
+          transition: 0.5s;
+          z-index: 1;
+          bottom: 80px;
+        }
+
+        .card .imgBx img {
+          position: absolute;
+          top: 0;
+          left: 0;
+          width: 100%;
+          height: 100%;
+          border-radius: 44px;
+          object-fit: cover;
+        }
+
+        .card .details {
+          position: absolute;
+          left: 10px;
+          right: 10px;
+          bottom: 10px;
+          height: 60px;
+          text-align: center;
+        }
+
+        .card .details h2 {
+          margin: 0;
+          padding: 0;
+          font-weight: 600;
+          font-size: 20px;
+          color: #000;
+          text-transform: uppercase;
+        }
+
+        .card .details h2 span {
+          font-weight: 500;
+          font-size: 16px;
+          color: #000;
+          display: block;
+          margin-top: 5px;
+        }
+        .color {
+          background-color: #84a98c;
+          width: 10px;
+          border-radius: 3px 0 0 3px;
+          height: 38px;
+          margin-right: 50px;
+        }
+        .info {
+          width: 300px;
+          border-radius: 2px;
+          height: 38px;
+          background-color: #cad2c5;
+          display: flex;
+          align-items: center;
+        }
+      }
+    </style>
```

Comment:
index.html is not the best place to add CSS.
Try to use tailwind classes whenever possible.